lib/x86/cpu_features.c

```diff
@@ -30,6 +30,27 @@
 
 #if HAVE_DYNAMIC_X86_CPU_FEATURES
```

Comment:
Isn't it necessary to make `HAVE_DYNAMIC_X86_CPU_FEATURES` and `HAVE_DYNAMIC_ARM_CPU_FEATURES` be defined when `_MSC_VER`?  Otherwise this code is never built.

Reply:
for now it's not built. Other chnages will enable this code.